NEWS.md

```diff
@@ -7,18 +7,31 @@
 * getResamplingIndices(inner = TRUE) now correctly returns the inner indices (before inner indices referred to the subset of the respective outer level train set)
 
 ## filter - new
-* praznik.JMI
-* praznik.DISR
-* praznik.JMIM
-* praznik.MIM
-* praznik.NJMIM
-* praznik.MRMR
-* praznik.CMIM
+* praznik_JMI
+* praznik_DISR
+* praznik_JMIM
+* praznik_MIM
+* praznik_NJMIM
+* praznik_MRMR
+* praznik_CMIM
+* FSelectorRcpp_gain.ratio
+* FSelectorRcpp_information.gain
+* FSelectorRcpp_symuncert
+
+## filter - renamed
```

Comment:
Why was this heading added when all the actual renamings are documented further down under "filter - general"?

Reply:
Thanks for spotting, I corrected it :+1: